app/controllers/artists_controller.rb

```diff
@@ -28,7 +27,6 @@ def create
 
   def update
     @artist = Artist.find(params[:id])
-
     if @artist.update(artist_params)
       flash.notice = "Arist '#{@artist.name}' Updated!"
```

Comment:
@s-espinosa @carmer should flash messages go somewhere else other than the controller?